ipywidgets/static/notebook/js/manager.js

```diff
@@ -79,7 +78,7 @@ define([
                         return WidgetManager._load_callback.call(that);
                     }).then(function(state) {
                         that.set_state(state);
-                    }).catch(utils.reject('Error loading widget manager state', true));
+                    }).catch(widgets.reject('Error loading widget manager state', true));
```

Comment:
reject should be a function of utils, it's designed to chain promise rejections IIRC